src/cache-redis/cache-redis.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@nestjs/common";
+import axios from "axios";
```

Comment:
You already installed `@nestjs/axios` why do you need to import it here again. Inject the `HttpModule` and use it there.